user-access-commons/src/main/java/no/unit/nva/model/UserList.java

```diff
@@ -12,6 +12,7 @@
 
     private final List<UserDto> users;
```

Comment:
@norrs pointed out to me that it is bad practice to return anything other than object in responses (no lists), and the response can not include jsonld context in the future in its current form.

Reply:
API objects should always be a json object, because of extensibility of the API without breaking API. See https://opensource.zalando.com/restful-api-guidelines/#110

Reply:
Should we create a bug for this?